server/backend/api/app/schemas.py

```diff
@@ -88,6 +88,13 @@ class FamRoleCreate(BaseModel):
     class Config:
         orm_mode = True
 
+class FamApplicationRoles(FamRoleCreate):
```

Comment:
Wondering... you probably meant to use the name: "FamApplicationRole" without "s"?

Reply:
Yep... That's now resolved in latest push

Reply:
Resolved this in latest push: